platform/ios/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash when switching between two styles having layers with the same identifier but different layer types. ([#12432](https://github.com/mapbox/mapbox-gl-native/issues/12432))
 * Fixed an issue where the symbols for `MGLMapPointForCoordinate` could not be found. ([#12445](https://github.com/mapbox/mapbox-gl-native/issues/12445))
 * Fixed an issue causing country and ocean labels to disappear after calling `-[MGLStyle localizeLabelsIntoLocale:]` when the system language is set to Simplified Chinese. ([#12164](https://github.com/mapbox/mapbox-gl-native/issues/12164))
+* Fixed a `CLLocationManager` crash when `MGLMapView` is deallocated. ([#12542](https://github.com/mapbox/mapbox-gl-native/pull/12542))
```

Comment:
> Fixed a crash that occurred when `MMELocationManager` was deallocated and the delegate was reporting updates. ([#12542](https://github.com/mapbox/mapbox-gl-native/pull/12542))

I think most of the crash reports mentions `MMELocationManager` and it may be more clear that this is their fix.

What do you think?

Reply:
Good point. @rclee can you tweak please?

Reply:
That's true for the majority. That makes sense -- thanks for the suggestion!